### DISCUSSION TITLE
How to set the unit of the temperature?

### DISCUSSION THREAD

**Discussion by coffeepublic** [artisan-roaster-scope/artisan#1023] [Q&A]
![16698964928387037761304423007548](https://user-images.githubusercontent.com/50404336/205049198-e3a8ab10-e329-49ca-804a-d156e8a2aaa8.jpg)

The temperature is showed by one decimal place. But the actual temperature are 52 and 74, not 5.2 and 7.4.  How can i adjust it ?

**Comment by MAKOMO** [comment]
Which roaster is connected? Which Artisan setup are you using? Could you please save your Artisan settings into an artisan-settings.aset file via menu Help >> Save Settings, rename that file into artisan-settings.txt and then drag-and-drop this on an answer field hier on the Github page?

Thanks!

**Reply by coffeepublic** [reply]
Proaster 0.5kg. Will do it when go to office. Thank you very much!

**Reply by bigl00z3** [reply]
Looking at Proaster config, it looks like it's using **Modbus**. 
Have a look at the divider value. You can change it in `Config>>Port.. ` then go to the **Modbus** tab and you will see the **Divider** value for each Input.

<img width="601" alt="image" src="https://user-images.githubusercontent.com/8886922/205231238-759b4256-2d3c-4e68-b132-94140a1ea340.png">

Nico

**Reply by MAKOMO** [reply]
Seems your machine has only 2 temperature channels thus you should deactivate the Input 3 as in the picture of Nico by setting its Slave to 0.

**Reply by coffeepublic** [reply]
> Which roaster is connected? Which Artisan setup are you using? Could you please save your Artisan settings into an artisan-settings.aset file via menu Help >> Save Settings, rename that file into artisan-settings.txt and then drag-and-drop this on an answer field hier on the Github page?
> 
> Thanks!

![IMG-20221203-WA0023](https://user-images.githubusercontent.com/50404336/205424280-7a446cd6-9d4e-46a2-b354-04b59123d9bc.jpg)
Attached is the ports of configuration

**Reply by coffeepublic** [reply]
> Looking at Proaster config, it looks like it's using **Modbus**. Have a look at the divider value. You can change it in `Config>>Port.. ` then go to the **Modbus** tab and you will see the **Divider** value for each Input.
> 
> <img alt="image" width="601" src="https://user-images.githubusercontent.com/8886922/205231238-759b4256-2d3c-4e68-b132-94140a1ea340.png">
> 
> Nico

Thanks for your help. I change dividers to blank, then it works!!
